### PR TITLE
Regular users can enable feature flag for receipt uploads

### PR DESCRIPTION
**Pull request by maxwofford**
Part of #2557. This should probably only be merged in after https://github.com/hackclub/bank/pull/2567, which adds permissions checking for receipt uploads.

### CODE REVIEW

**Comment by maxwofford** [comment]
The requested review here is less for the line changes in this PR, more for a sanity check before releasing this on the public. It doesn't need to be good yet, it just needs to work well enough for it to be a feature preview. I'll merge once there are review approvals.

**Review by cjdenio — APPROVED** [review]
This looks great to me!

**Review by exu3 — APPROVED** [review]
Looks good to me! Can confirm the feature works well.

**Review by garyhtou — APPROVED** [review]
good to go!

File: app/views/users/edit.html.erb
Comment (garyhtou) [inline app/views/users/edit.html.erb]: ~~should this be disabled? or just be text?~~

nvm! i see its commented out

Comment (maxwofford) [reply app/views/users/edit.html.erb]: This is commented out for the time being while getting some styling correct. It's kinda weird, but https://github.com/hackclub/bank/pull/2566/files#diff-7a664d6e0b1d39425223431cffbf717843cbac2e33ac4fe6a58ecb959e38f6abR104 is a check to see if the feature is generally available, and doesn't check for the current user.

**Pull request merged**
